client/app/catalog/catalog.html

```diff
@@ -1,96 +1,163 @@
-<div id="sidebar-wrapper">
-  <ul class="sidebar-nav">
-    <li class="sidebar-brand">
-      <a href="#">Filter Catalog</a>
-    </li>
-    <li>
-      <a href="#">Suggestion # 1</a>
-    </li>
-    <li>
-      <a href="#">Suggestion # 2</a>
-    </li>
-    <li>
-      <a href="#">Suggestion # 3</a>
-    </li>
-    <li>
-      <a href="#">Suggestion # 4</a>
-    </li>
-    <li>
-      <a href="#">Suggestion # 5</a>
-    </li>
-    <li>
-      <a href="#">Suggestion # 6</a>
-    </li>
-    <li>
-      <a href="#">Suggestion # 7</a>
-    </li>
-  </ul>
-</div>
-<div class="container-fluid">
-  <div id="catalog-photos" class="col-md-8 col-md-offset-3">
+<div class="row-fluid">
+  
+  <!-- Sidebar wrapper -->
+  <div id="sidebar-wrapper" class="col-sm-2"s>
+
+    <ul class="sidebar-nav">
+      <li class="sidebar-brand">
+        Filter Catalog
+      </li>
+      <li>
+        <a href="#">Suggestion # 1</a>
+      </li>
+      <li>
+        <a href="#">Suggestion # 2</a>
+      </li>
+      <li>
+        <a href="#">Suggestion # 3</a>
+      </li>
+      <li>
+        <a href="#">Suggestion # 4</a>
+      </li>
+      <li>
+        <a href="#">Suggestion # 5</a>
+      </li>
+      <li>
+        <a href="#">Suggestion # 6</a>
+      </li>
+      <li>
+        <a href="#">Suggestion # 7</a>
+      </li>
+    </ul>
+  </div>
+  <!-- End sidebar wrapper -->
+
+  <!-- Start photo catalog -->
+  <div id="catalog-photos" class="col-sm-10">
     <div class="row">
-      <div class="col-md-4">
+
+      <!-- Product info -->
```

Comment:
Same comment as above - repeated html elements should be replaced with ng-repeat="[1,2...n]"